File: app/src/main/java/com/example/vest/sdk/app/AppTestApplication.java
package com.example.vest.sdk.app;

import androidx.multidex.MultiDexApplication;

import code.sdk.core.VestSDK;


public class AppTestApplication extends MultiDexApplication {

    @Override
    public void onCreate() {
        super.onCreate();
        VestSDK.setLoggable(true);
        VestSDK.init(getBaseContext(), "config");
    }

}

File: app/src/main/java/com/example/vest/sdk/app/AppTestApplication.kt
package com.example.vest.sdk.app

import androidx.multidex.MultiDexApplication
import code.sdk.core.VestSDK

class AppTestApplication : MultiDexApplication() {
    override fun onCreate() {
        super.onCreate()
        VestSDK.setLoggable(true)
        VestSDK.init(baseContext, "config")
    }
}